psalm-baseline.xml

```diff
@@ -823,6 +823,9 @@
       <code>joinColumnName</code>
       <code>joinColumnName</code>
     </TooManyArguments>
+    <UndefinedMethod occurrences="1">
+      <code>isReadOnly</code>
+    </UndefinedMethod>
```

Comment:
Looks like you ran Psalm with PHP < 8.1

Reply:
Just a moment, please. I need to reinstall PHP on my environment. :/

Reply:
Updated.